dpdata/vasp/outcar.py

```
@@ -6,31 +6,94 @@
+    nelm: optional[int]
+        the value of NELM parameter
+    nwrite: optional[int]
+        the value of NWRITE parameter
+    """
     atom_names = []
+    atom_names_potcar = []
     atom_numbs = None
     nelm = None
+    nwrite = None
     for ii in lines:
         ii_word_list = ii.split()
```

Ruff (0.11.9)

Local variable `ii_word_list` is assigned to but never used

Remove assignment to unused variable `ii_word_list`

(F841)